cli/internal/fs/turbo_json_test.go

```diff
@@ -29,15 +29,15 @@ func Test_ReadTurboConfig(t *testing.T) {
 
 	pipelineExpected := map[string]TaskDefinition{
 		"build": {
-			Outputs:                 []string{"dist/**", ".next/**"},
+			Outputs:                 TaskOutputs{Inclusions: []string{"dist/**", ".next/**"}, Exclusions: []string{"dist/assets/**"}},
```

Comment:
Can you update `validatePipeline` to ensure that these remain sorted?

Reply:
I don't see a `validatePipeline` function. There is a generic `validateOutput` function but that just does a comparison between the expected result and the actual result. Do you want me to check if the actual result has sorted fields?